src/ray/raylet/local_task_manager.cc

```diff
@@ -55,27 +55,26 @@ LocalTaskManager::LocalTaskManager(
     size_t max_pinned_task_arguments_bytes,
     std::function<int64_t(void)> get_time_ms,
     int64_t sched_cls_cap_interval_ms)
-    : self_node_id_(self_node_id),
+    : self_node_id_(std::move(self_node_id)),
       cluster_resource_scheduler_(cluster_resource_scheduler),
       task_dependency_manager_(task_dependency_manager),
       is_owner_alive_(is_owner_alive),
-      get_node_info_(get_node_info),
+      get_node_info_(std::move(get_node_info)),
       max_resource_shapes_per_load_report_(
           RayConfig::instance().max_resource_shapes_per_load_report()),
       worker_pool_(worker_pool),
       leased_workers_(leased_workers),
-      get_task_arguments_(get_task_arguments),
+      get_task_arguments_(std::move(get_task_arguments)),
       max_pinned_task_arguments_bytes_(max_pinned_task_arguments_bytes),
-      get_time_ms_(get_time_ms),
+      get_time_ms_(std::move(get_time_ms)),
       sched_cls_cap_enabled_(RayConfig::instance().worker_cap_enabled()),
       sched_cls_cap_interval_ms_(sched_cls_cap_interval_ms),
       sched_cls_cap_max_ms_(RayConfig::instance().worker_cap_max_backoff_delay_ms()) {}
 
 void LocalTaskManager::QueueAndScheduleTask(std::shared_ptr<internal::Work> work) {
   // If the local node is draining, the cluster task manager will
   // guarantee that the local node is not selected for scheduling.
-  ASSERT_FALSE(
```

Comment:
which `#import` did this macro get in from? Shall we also remove that macro, and maybe in bazel target deps?

Reply:
For our build file organization, it's not easy to check by bazel target dependency, since we place all files into one giant target.
https://github.com/ray-project/ray/blob/2dd606198c83ccc86c4dbbef5f59499041e85ba8/BUILD.bazel#L675-L695

Reply:
sad news